src/main/java/org/alliancegenome/curation_api/jobs/executors/ParalogyExecutor.java

```diff
@@ -62,4 +67,30 @@ public void execLoad(BulkLoadFile bulkLoadFile) {
 		}
 	}
 
+	private void runCleanup(BulkLoadFileHistory history, String dataProvider, List<Long> paralogyPairsBefore, List<Long> paralogyPairsAfter) {
```

Comment:
@VarunReddy1111 can the generic cleanup be used or do we need this specific one?